packages/sdk/src/wallet/providers/DefaultSmartWalletProvider.ts

```diff
@@ -30,7 +30,7 @@ export class DefaultSmartWalletProvider extends SmartWalletProvider {
    */
   constructor(
     chainManager: ChainManager,
-    lendProvider: LendProvider,
+    lendProvider?: LendProvider,
```

Comment:
I'm not sure how the WalletProvider <> LendProvider interface should work in the end